app/celery/tasks.py

```diff
@@ -298,12 +303,11 @@ def save_smss(self, service_id: Optional[str], signed_notifications: List[Signed
 
     # we should move this check inside the for loop below
     check_service_over_daily_message_limit(KEY_TYPE_NORMAL, service)
-
     current_app.logger.info(f"Sending following sms notifications to AWS: {notification_id_queue.keys()}")
     for notification_obj in saved_notifications:
         try:
-            check_if_request_would_put_service_over_daily_sms_limit(KEY_TYPE_NORMAL, service, 0)  # type: ignore
-        except LiveServiceTooManySMSRequestsError:
+            check_service_over_daily_sms_limit(KEY_TYPE_NORMAL, service)
```

Comment:
Great simple fix. With the goal in mind to avoid potential confusion like this one, could we rename the function to indicate a send might happen so this is clearer? Something like `check_service_over_daily_sms_limit_and_send`? That would be the quick and easy confusion fix.

It looks to me though the long-term ideal fix would be to decouple the checks from the sending though, to have one concern per function (1 function check and 1 function send). The sending function could then be called in the except clause.

Reply:
> It looks to me though the long-term ideal fix would be to decouple the checks from the sending
Yes I agree - that would make it much clearer

> could we rename the function to indicate a send might happen so this is clearer
Good idea, I'll make that change now

Reply:
I renamed to `check_service_over_daily_sms_limit_and_warn` since that is the language used in some of the other functions like `warn_about_daily_sms_limit`.